app/networkapi/templates/shared/adminsortable_objects_custom.html

```diff
@@ -4,7 +4,7 @@
     <input name="pk" type="hidden" value="{{ object.pk }}" />
     <div id="sortable-object">
         <a href="{% url opts|admin_urlname:'do_sorting' object.model_type_id %}" class="admin_sorting_url">
-            <img class="person-image" src="{{ object.image.url }}">
+            <img class="instance-image" src="{{ object.image.url }}">
```

Comment:
I'll keep the class name generic, so we can share CSS, but we'll be able to customize the src string.